### PR TITLE
Use all hyperblock query params

### PR DESCRIPTION
**Pull request by mariusmihaic**
- Added several hyper block query params which have been added meanwhile:
```
type HyperBlockQueryOptions struct {
	WithAlteredAccounts bool   `toml:"withLogs"`
	NotarizedAtSource   bool   `toml:"notarizedAtSource"`
	Tokens              string `toml:"tokens"`
	WithMetaData        bool   `toml:"withMetaData"`
}
```
- Moved all query params in `config.toml`. Proxy will load them and use them for each query
- Changed `Hash` field from `MetaData` to be `bytes` instead of `fixed` size, since it can have any length.

### CODE REVIEW

File: cmd/proxy/config/config.go [hunk 1]
```diff
@@ -8,10 +8,19 @@ import (
...
+	ElrondProxyUrl         string                 `toml:"elrondProxyUrl"`
+	RequestTimeOutSec      uint64                 `toml:"requestTimeOutSec"`
+	HyperBlockQueryOptions HyperBlockQueryOptions `toml:"hyperBlockQueryOptions"`
+}
+
+type HyperBlockQueryOptions struct {
+	WithLogs            bool   `toml:"withLogs"`
+	WithAlteredAccounts bool   `toml:"withLogs"`
+	NotarizedAtSource   bool   `toml:"notarizedAtSource"`
+	Tokens              string `toml:"tokens"`
```
Comment (bogdan-rosianu) [inline cmd/proxy/config/config.go]: I think this configuration holder file can be held under `covalent-indexer-go/config` directory, in accordance with other projects

Comment (miiu96) [inline cmd/proxy/config/config.go]: sort variable types... all the `bool` types should stay together

Comment (mariusmihaic) [reply cmd/proxy/config/config.go]: Used the same pattern from `tools` repository. Also, this config is specific to the loaded config, so I would leave it as it is

Comment (mariusmihaic) [reply cmd/proxy/config/config.go]: Refactored

**Review by miiu96 — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**